src/display/annotation_layer.js

```diff
@@ -2611,7 +2613,6 @@ class AnnotationLayer {
         enableScripting: parameters.enableScripting,
         hasJSActions: parameters.hasJSActions,
         fieldObjects: parameters.fieldObjects,
-        mouseState: parameters.mouseState || { isDown: false },
```

Comment:
I'm not at all sad to see that parameter go!

However there's still some `mouseState` occurrences found in this file, in both code and JSDocs, so please go through and remove those as well.

Reply:
You know what ??.... me neither ;)